aspnet/mvc/views/forms/sample/final/Views/Demo/EditColor.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@model Person
+@{
+    var index = (int)ViewData["index"];
+}
+
+<form asp-controller="ToDo" asp-action="Edit" method="post">
+    @*
+        Move the following two lines to a template to avoid
+        evaluating index twice.
+        <label asp-for="Colors[index]" />
+        <input asp-for="Colors[index]" /> <br />
+    *@
```

Comment:
Remove this comment. It duplicates information covered elsewhere in the document and talks about a change you've made.